### ISSUE TITLE
A11y_OSPO Production_Projects_Contrast : The color contrast ratio for "CNAB", "Accessibility Insights" and "Windows Terminal" links is 3.45:1 which is less than 4.5:1

### ISSUE THREAD

**Issue by Rachita98** [microsoft/opensource.microsoft.com#121]
<h2>GitHub Tags:</h2>
#A11yTCS;#A11yMAS;#Win10;#DesktopWeb;#ChromiumEgde;#A11ySev2;#P2_WCP;#WCP;#P2_OSPOProduction_Web_Sep2021;#P2_OSPOProduction;#OSPO Production;#WCAG1.4.3;#Benchmark;#Color Contrast;

<h2>Environment Details:</h2>
URL: https://opensource.microsoft.com/

<h2> OS and Browser Details:</h2>
Microsoft Windows 10 Enterprise Version 21H1 (OS build 19043.1165)
Microsoft Edge Version 95.0.1000.0 (Official build) dev (64-bit)

<h3>Note:</h3>
This issue is also observed in below links in the "Open Source Updates" and the associated "Read post" links present there:
1. Microsoft joins Open Source Security Foundation
2. Helm, the package manager for Kubernetes, now a CNCF graduated project
3. Microsoft announces OpenChain 2.0 conformance for open source
4. exFAT in the Linux kernel? Yes!

<h2> Repro Steps:</h2>
1. Hit the URL:https://opensource.microsoft.com/
2. Check whether the color contrast ratio for "CNAB", "Accessibility Insights" and "Windows Terminal" links is 3.45:1

<h2>Actual Result:</h2>
The color contrast ratio for "CNAB", "Accessibility Insights" and "Windows Terminal" links is 3.45:1 which is less than 4.5:1

<h2>Expected Result:</h2>
The color contrast ratio for "CNAB", "Accessibility Insights" and "Windows Terminal" links should be 4.5:1 or more than that.

<h2>User Impact:</h2>
Users with low vision may not be able to identify link if the color contrast ratio for "CNAB", "Accessibility Insights" and "Windows Terminal" links is less than 4.5:1.

<h2>Attachments for reference:</h2>

![#121_Contrast_The color contrast ratio fails for CNAB, Accessibility Insights and Windows Terminal links](https://user-images.githubusercontent.com/90394513/132861276-3187a7a3-a037-4029-ac13-de63b53c7be9.png)

<h2>Note Section Attachment:</h2>

![#121_Note Section](https://user-images.githubusercontent.com/90394513/132861349-56972432-153e-4654-b7c6-247423bd4d6b.png)

**Comment by ferBonnin** [comment]
per the [Understanding doc for 1.4.3 Contrast (minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html), large text (at least 18pts or 14pt bold) must have a contrast of at least 3:1.  The text highlighted in the first screenshot is 24px. The _read post_ text (foreground color #1C77BC, background color #FFFFFF) has a Ratio of 4.757:1

@Rachita98 can you please clarify more if there is a bug here?

cc @jeffwilcox

**Comment by Rachita98** [comment]
Closing this bug as per SME's approval.

**Comment by AnKushSingh05** [comment]
Adding Closed Tag
